dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/BroadcastClusterInvoker.java

```diff
@@ -55,6 +62,7 @@ public Result doInvoke(final Invocation invocation, List<Invoker<T>> invokers, L
         // 100 means that an exception will be thrown last, and 0 means that as long as an exception occurs, it will be thrown.
         // see https://github.com/apache/dubbo/pull/7174
         int broadcastFailPercent = url.getParameter(BROADCAST_FAIL_PERCENT_KEY, MAX_BROADCAST_FAIL_PERCENT);
+        boolean collectAllResults = url.getParameter(BROADCAST_COLLECT_ALL_RESULT_KEY, false);

```

Comment:
Please add some comment about this parameter

Reply:
get